telegram/_bot.py

```diff
@@ -4271,7 +4271,7 @@ async def get_updates(
         self,
         offset: Optional[int] = None,
         limit: Optional[int] = None,
-        timeout: Optional[int] = None,
+        timeout: Optional[int] = None,  # noqa: ASYNC109
```

Comment:
What's this about?

Reply:
it tells to use Asyncio.timeout which is wrong here ofc